File: TaskManager/ClientApp/app/components/shared/models/projectfile.ts
export class ProjectFile {
	projectFileID: number;
	projectID: number;
	fileName: string;
	isAdded: boolean;
	isDeleted: boolean;
	fileData: any;

	constructor() {
		this.projectFileID = 0;
		this.projectID = 0;
		this.fileName = "";
		this.isAdded = false;
		this.isDeleted = false;
	}
}

export interface IOneProjectFile {
	projectFile: ProjectFile;
}

File: TaskManager/Models/ProjectFile.cs
using System.Collections.Generic;

namespace TaskManager.Models
{
	public class ProjectFile
	{
		public int ProjectFileID { get; set; }
		public int ProjectID { get; set; }
		public string FileName { get; set; }
	}

	public class ProjectFileSend : ProjectFile
	{
		public bool IsAdded { get; set; }
		public bool IsDeleted { get; set; }
	}

	public class ProjectFilesList
	{
		public List<ProjectFileSend> projectFiles { get; set; }
	}

}
